Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1648,6 +1648,22 @@ void HTMLInputElement::clear_algorithm()
     update_shadow_tree();
 }
 
+void HTMLInputElement::post_connection()
```

Comment:
I'm not convinced post connection steps are technically correct and there is some subtle way it would be broken, otherwise the spec when this would have been introduced would have used this hook.

If we followed spec to the letter, this should be inside the inserted() callback inside a is_connected() if statement.

Reply:
e.g one way this is observable is that script can run inside of the post connection steps, so i beleive the timing of the change would probably be observably different